### PR TITLE
Amj/scriptscanner fixes

### PR DESCRIPTION
**Pull request by jayich**
review: @aransfor , @gregllong 

Please take a look.  Our experiments seem to run fine.

### CODE REVIEW

**Comment by aransfor** [comment]
@jayich  Just to clarify, there are a few classes in this file, https://github.com/labrad/pylabrad/blob/master/labrad/server.py One is labradserver and one is signal(which is quite short). I dont see one inheriting from the other so how can scriptscanner have all the functionality of a server if it doesnt inherit the server as well. Maybe Im confused on what you mean by subclass

**Comment by jayich** [comment]
ScriptScanner doesn't inherit Signal from pylabrad, it inherits Signals from inside common.  I changed the name in https://github.com/CampbellGroup/common/pull/89/commits/6afffa035e5a02c944bf79955e4f8a041231422c because this is confusing.  Signals is a base server class of ScriptScanner.  The functionality probably should not be in this server and inherited, but put directly into ScriptScanner.

File: lib/servers/abstractservers/script_scanner/script_scanner.py [hunk 1]
```diff
@@ -90,44 +88,45 @@ def load_scripts(self):
                 else:
                     self.script_parameters[name] = script_class_parameters(name, cls, parameters)
 
-    @setting(0, "Get Available Scripts", returns = '*s')
+    @setting(0, "get_available_scripts", returns='*s')
```
Comment (gregllong) [inline lib/servers/abstractservers/script_scanner/script_scanner.py]: Will clients running the script_scanner server still be able to use the same method name with this change?

Comment (gregllong) [inline lib/servers/abstractservers/script_scanner/script_scanner.py]: Alright sure it's a parsing change. Ignore my last comment.

**Comment by gregllong** [comment]
@aransfor I think the idea is that [Signals](https://github.com/CampbellGroup/common/blob/amj/scriptscanner_fixes/lib/servers/abstractservers/script_scanner/script_signals_server.py#L4) is a child class of LabradServer, which means inheriting from Signals alone would make ScriptScanner a legit LabradServer. The [Signal](https://github.com/labrad/pylabrad/blob/master/labrad/server.py#L37) object you pointed to is not the same object.

**Comment by gregllong** [comment]
LGTM

**Comment by aransfor** [comment]
LGTM

**Pull request merged**